src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/CustomMarshallerAttributeFixerTests_StatelessValueShapeValidation.cs

```diff
@@ -376,150 +376,7 @@ public static int BufferSize
             await VerifyCS.VerifyCodeFixAsync(
                 source,
                 fixedSource,
-                VerifyCS.Diagnostic(CallerAllocConstructorMustHaveBufferSizeRule).WithLocation(0).WithArguments("MarshallerType", "byte"));
-        }
-
-        [Fact]
-        public async Task ModeThatUsesManagedToUnmanagedShape_Missing_ConvertToUnmanagedMethod_Marshaller_DifferentDocument_ReportsDiagnostic()
```

Comment:
Why are these deleted?

Reply:
I think that was a bad merge. I'll make sure to bring these back